### ISSUE TITLE
Should we have a red team on these projects?

### ISSUE THREAD

**Issue by linneagandhi** [Watts-Lab/nudge-cartography#61]

**Comment by JamesPHoughton** [comment]
In engineering we had "Preliminary Design Reviews" and "Final Design Reviews" at strongly standardized points in the process. They would normally take most of a day, and would have different experts do a super deep dive on the technical content. They would serve as gates that you would have to pass before you spend money doing detailed engineering (preliminary) and then cutting metal (final design review).

**Comment by markwhiting** [comment]
I agree that we need a review and check cycle and other related things. I am not sure I think it needs to be an extra review process every time, or if a checklist for most points would be sufficient. Even a check list that is sketched out for a project some time early on, with goals that need to be checked at future points could be a great option. 

In any case, I like all of this, but agree with Duncan's sentiment that a meeting & human intensive approach might be too heavy. So perhaps more chances to show concrete work and more changes to force our selves to check our assumptions would be a good start.

**Comment by JamesPHoughton** [comment]
Agree that more tests of our work earlier is better, in almost every case. Figuring out what those tests are is time well invested. Internal tests of a designed platform are a must. Pilots with real humans also. Would be good to see what we can use as tests earlier in the process. Maybe mockups with the intended stimuli, where we sortof run a minimum viable analog(ue) before we build new code.

**Comment by markwhiting** [comment]
+1. 

Perhaps also iterative writing, like trying to write up bits of the paper as we go, so a lot of the challenges of converting ideas into practice get critically assessed at earlier points.

**Comment by markwhiting** [comment]
@linneagandhi could we move it to https://github.com/Watts-Lab/projects perhaps?

That is internal (not public to humanity) but visible by everyone here.
